baseband/guppi/frame.py

```diff
@@ -0,0 +1,105 @@
+# Licensed under the GPLv3 - see LICENSE
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..vlbi_base.frame import VLBIFrameBase
+from .header import GUPPIHeader
+from .payload import GUPPIPayload
+
+
+__all__ = ['GUPPIFrame']
+
+
+class GUPPIFrame(VLBIFrameBase):
+    """Representation of a GUPPI file, consisting of a header and payload.
+
+    Parameters
+    ----------
+    header : `~baseband.dada.GUPPIHeader`
```

Comment:
link wrong `baseband.guppi` not `baseband.dada`... (happens a few more times...)

Reply:
Fixed them all, hopefully.